modules/appeals_api/spec/factories/higher_level_reviews.rb

```diff
@@ -6,25 +6,29 @@
     id { SecureRandom.uuid }
     api_version { 'V1' }
     auth_headers do
-      JSON.parse(File
-        .read(::Rails.root.join(*'/modules/appeals_api/spec/fixtures/v1/valid_200996_headers.json'.split('/')).to_s))
+      JSON.parse(File.read(::Rails.root.join(
```

Comment:
The way these attributes get populated from a fixture file is pretty esoteric. I know you were just updating the path & some simple formatting changes, but that's just putting lipstick on a pig. At minimum these should use the Engine root instead of Rails root, and I'm sure there's more that could be done to make this cleaner. Maybe another usecase for FixtureHelpers? Or maybe just define a local method/lambda to extract this repeated code out?

If you see an 🍎 , then this same comment applies to that file, too. :P

Reply:
Thanks, I'll take a look at this for my next PR!

Reply:
I ended up giving these changes their own PR: https://github.com/department-of-veterans-affairs/vets-api/pull/13001